server/core/include/irods/configuration_hook_manager.hpp

```diff
@@ -0,0 +1,65 @@
+#ifndef CONFIGURATION_HOOK_MANAGER_HPP
+#define CONFIGURATION_HOOK_MANAGER_HPP
+
+#include <vector>
+#include <string>
+#include <nlohmann/json.hpp>
+/// \file
```

Comment:
This directive should be placed above `#include` directives to match other iRODS files.